File: week1/Bu$ra/ContainsDuplicate.cs
using System;
using System.Collections.Generic;

public class Solution {
    public static bool ContainsDuplicate(int[] nums) {
        HashSet<int> set = new HashSet<int>();

        foreach (int num in nums)
        {
            if (!set.Add(num))
            {
                return true;
            }
        }

        return false;
    }

File: week1/Emopusta (C# && C++)/C++/containsDuplicate.cpp
#include <unordered_set>
#include <vector>

using namespace std;

class Solution {
public:
    bool containsDuplicate(vector<int>& nums) {
        unordered_set<int> set;
        for (int num : nums)
        {
            set.insert(num);
        }
        return set.size() != nums.size();
    }
};